.golangci.yml

```diff
@@ -9,42 +9,18 @@ linters:
   disable-all: true
   enable:
     - lll
-    - misspell
     - gofumpt
-    - govet
     - deadcode
     - goimports
     - ineffassign
-    - varcheck
-    - errcheck
-    - revive
-    - gosec
-    - gosimple
+    - misspell
 
 linters-settings:
   lll:
-    line-length: 150
-  maligned:
-    suggest-new: true
+    line-length: 261
```

Comment:
Is this the final spot we want it? If not, might want to put a note/todo as to the desired final length

Reply:
Mostly I found the length that doesn't report any errors.  If I go to 150 I will have to change alot of code.  even 200 has a 10s of files that I should change. Figured could put a upper limit and fail if anything is greater.  We have 1 file that has 260 line length

Reply:
This works :) I was just thinking, in the future, it might be a low hanging fruit for someone to get this lowered to some point, if we agree on the end point (e.g. 150 characters?). Doesn't really matter now :)